key/group.go

```diff
@@ -227,7 +233,7 @@ func (g *Group) FromTOML(i interface{}) (err error) {
 
 	if gt.PublicKey != nil {
 		// dist key only if dkg ran
-		g.PublicKey = &DistPublic{}
+		g.PublicKey = &DistPublic{Scheme: *sch}
```

Comment:
This really feels hackish and ugly, to have to monitor for all usages of DistPublic and make sure the scheme is set...

Reply:
perhaps you could make `s/DistPublic/distPublic` and control creation with a `NewDistPublic(scheme Scheme)` func

Reply:
ah wait maybe it lives in Kyber though